mara_db/shell.py

```diff
@@ -5,10 +5,11 @@
 """
 
 import shlex
-import sys
 from functools import singledispatch
 
+import sys
 from mara_db import dbs, config
+from mara_db.bigquery import bigquery_credentials
```

Comment:
Is this a patchable config? if so, `import mara_db.bigquery as bq` and then use `bq.bigquery_credentials`. This might otherwise break if this module for some reason is imported before the patching.